### PR TITLE
Adding section 2.2 and 2.4

### PR DESCRIPTION
**Pull request by moshonk**
@ssmusoke,  
Added section 2.2 and 2.4 (Split from  the 105Section1 PR)
cc @ningosi

### CODE REVIEW

**Comment by ningosi** [comment]
@ssmusoke please find time to look at this. Let us plan to clear this before we break for elections

File: api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java [hunk 1]
```diff
@@ -211,4 +213,317 @@ public CohortDefinition hasObsAndEncounter(String encounterType, Concept q, Conc
         cd.setCompositionString("ancEncounter AND hasObs");
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java]: @moshonk This is no longer ANC - so need to clean up the comments

**Comment by ssmusoke** [comment]
@moshonk Comments on Maternity:

1. M5 and M6 - return all patients tested and not only those in maternity, need to filter to the maternity encounter 

2. The following are not translated M.M11B, M.M14, M.M16

**Comment by moshonk** [comment]
@ssmusoke, I have addressed the issue raised.

**Comment by ssmusoke** [comment]
@moshonk M11 - 

1. Low birth weight for < 2.5 is not showing any data. Check use of setValue & setOperator2 vs setOperator1 & setValue1 (operator and value with similar prefix)

2. M2 Referrals to Maternity Unit - showing 0 for all data 

3. M4 - Double check the age disgregations ... 
- 10 - 19 means any one older than 10 and less than 20
- 20 - 24 older than 20 but  less than 25 
- 25+ - 25 and above

**Comment by ssmusoke** [comment]
@moshonk EID - E.E6 is not translated to a value

**Comment by moshonk** [comment]
For referrals to maternity Unit. Please refer to the maternity register which states that the REF Number field be populated with the 'REF' value if a woman is being referred
image

**Comment by moshonk** [comment]
For M2, I am using the following criteria to get **Women testing HIV+ in labour - 1st time this Pregnancy**.. 
EMTCTCode == TRRK (New clients with Known HIV+ status)
Is this correct? I may need your indulgence on this one.

**Comment by moshonk** [comment]
For M8, I am using the following:
**Pregnancy outcome == Any
HIV Status == HIV Positive**

Should I use eMTCT codes instead of the HIV Status field to check the HIV status?

**Comment by moshonk** [comment]
@ssmusoke About M4, the disaggregations look ok on my end. Whats  the exact scenario that you've encountered on your end?

**Comment by ssmusoke** [comment]
@moshonk Please resolve the conflicts in this PR due to recent commits

**Comment by moshonk** [comment]
@ssmusoke, conflicts resolved.

**Comment by ssmusoke** [comment]
@moshonk @ningosi Please can you look at this report for a patient aged 25 years, I have one who is not appearing in the report

**Comment by ningosi** [comment]
@ssmusoke  I will review and advice. Which month are you are running against and which patient is missing? are yo using the same db as that of smc?

**Comment by ningosi** [comment]
@ssmusoke did the patient appear on the list eventually per your concerns? **Please can you look at this report for a patient aged 25 years, I have one who is not appearing in the report**

**Comment by ningosi** [comment]
Check ci failure @moshonk

**Pull request merged**